src/StateLabel.tsx

```diff
@@ -38,7 +38,7 @@ const StateLabelBase = styled.span<StyledStateLabelBaseProps>`
   align-items: center;
```

Comment:
StateLabel has variants that will need to be moved into this file. You can use https://github.com/primer/css/blob/mkt/color-modes-whee/src/labels/states.scss as an implementation reference.

Here's documentation on how to create variants with styled system: https://styled-system.com/variants

Reply:
done!